app/wrappers/Executors.js

```diff
@@ -90,6 +90,12 @@ class Executors {
     executorsRemoved() {
         return this.executorsData.executorsRemoved || [];
     }
+
+    executorsAliasList() {
+        return this.executorsList
+            .filter(executor => executor.aliasReason === 'Change by deed poll' || executor.currentNameReason === 'Change by deed poll')
```

Comment:
Can we use the same key for both the main applicant and the other applicants rather than having `aliasReason` for one and `changeNameReason` for the other?

Reply:
As discussed, happy to change but due to the fact both values mean different thing for the main executor and the secondary executors could get confusing if they had the same value. Will leave as is for now, but definitely raise with Femi when he is back.

Reply:
Sure ok